docs/source/howto/archive_profile.md

```diff
@@ -20,69 +20,50 @@ This tutorial can be downloaded and run as a Jupyter Notebook: {nb-download}`arc
 ```
 
 The AiiDA archive is a file format for long term storage of data from a particular profile.
-
 See {ref}`how-to:share:archives` for information on how to create and migrate an archive.
-Once you have an archive at the latest version, you can inspect its contents in the same way you would with a standard AiiDA profile.
-
-We first create a profile instance from the archive path:
-
-```{code-cell} ipython3
-from aiida import manage, orm, profile_context
-from aiida.storage.sqlite_zip.backend import SqliteZipBackend
-
-archive_profile = SqliteZipBackend.create_profile('include/process.aiida')
-print(archive_profile)
-```
-
-The {py:func}`~aiida.manage.configuration.profile_context` function works similarly to the {py:func}`~aiida.manage.configuration.load_profile` function,
-but is used within a context manager, that insures that the storage is properly closed when the context is exited.
-With this, we can load our archive as a profile:
-
-```{code-cell} ipython3
-with profile_context(archive_profile):
-    print(manage.get_manager().get_profile())
-```
 
-To directly access the storage backend, and view information about it, we can use:
+The easiest way to inspect the contents of an archive is to create a profile that "mounts" the archive as its data storage:
 
 ```{code-cell} ipython3
-import json
-with profile_context(archive_profile):
-    storage = manage.get_manager().get_profile_storage()
-    print(storage)
-    print(json.dumps(storage.get_info(), indent=2))
+!verdi profile setup core.sqlite_zip -n --profile archive --filepath include/process.aiida
```

Comment:
```suggestion
!verdi profile setup core.sqlite_zip -n --profile archive --filepath process.aiida
```

I guess the rebase swallows this comment? :smile: But it is a minor one, feel free to ignore it.

Reply:
I think that will actually fail the build on the docs, would it not? It is actually running the md file as a notebook, and in the source, the archive is in the `include/process.aiida` relative path. It _is_ a bit annoying that if the user downloads the archive, they have to either update the path in the notebook or manually create the `include` folder and move it there. But maybe we can move the archive at the same level as the notebook in the docs. It is nicer to keep it separate as that is the structure for the rest of the docs, but it would make it user friendly and maybe that is more important here.

Reply:
I agree the user friendliness is more important, let's move the file.

Reply:
Done